### PR TITLE
[BPK-4101]: Refresh/fixes/new style Ratings

### PR DESCRIPTION
**Pull request by olliecurtis**
Screenshot 2020-07-28 at 14 51 51
Screenshot 2020-07-28 at 14 51 56

### CODE REVIEW

File: packages/bpk-component-rating/README.md [hunk 1]
```diff
@@ -31,12 +31,15 @@ export default () => (
 | title     | string                | true     | -                 |
 | value     | number                | true     | -                 |
 | className | string                | false    | null              |
+| pill      | boolean               | false    | false             |
```
Comment (shaundon) [inline packages/bpk-component-rating/README.md:34]: An enum is preferable for styling properties, as it leaves the API open for any future styles that need to be added. 

Consider if in future we have to add a 'square' type. If we use a boolean we'd be stuck, but if we have a `RATING_TYPE` enum we can just add a new type.

File: packages/bpk-component-rating/src/BpkRating-test.js [hunk 2]
```diff
@@ -181,4 +181,31 @@ describe('BpkRating', () => {
...
   });
+
+  it('should render correctly pill rating', () => {
+    const tree = renderer
+      .create(
+        <BpkRating
+          ariaLabel="6.7 Average might recommend"
+          title="Average"
+          subtitle="Might recommend"
+          value={6.7}
```
Comment (shaundon) [inline packages/bpk-component-rating/src/BpkRating-test.js:192]: These tests seem to be missing the `pill` prop?

File: packages/bpk-component-rating/src/common-types.js [hunk 3]
```diff
@@ -18,6 +18,9 @@
 
 /* @flow */
 
-const RATING_SIZES = { sm: 'sm', base: 'base', lg: 'lg' };
+export const RATING_SIZES = { sm: 'sm', base: 'base', lg: 'lg' };
 
-export default RATING_SIZES;
+export const RATING_TYPE = {
```
Comment (shaundon) [inline packages/bpk-component-rating/src/common-types.js:23]: Can it be plural (`RATING_TYPES`) to match our convention of using plural elsewhere? 😇

File: packages/bpk-component-rating/README.md (hunk 1 above)
Comment (shaundon) [inline packages/bpk-component-rating/README.md:34]: Readme needs updating to use `RATING_TYPES` instead of `pill`.

Comment (olliecurtis) [reply packages/bpk-component-rating/README.md:34]: Yep just realised after I requested review 😆

File: UNRELEASED.yaml [hunk 4]
```diff
@@ -1,6 +1,12 @@
 # UNRELEASED:
 
-# How to write a good changelog entry:
+ADDED:
+  - bpk-component-rating:
+    - Rating has been designed to have a fresh look.
+    - Fixed A11y contrast issues with `high` and `medium` ratings.
+    - Added new `pill` style rating which is set with the `pill` property.
```
Comment (shaundon) [inline UNRELEASED.yaml:7]:
```suggestion
    - Added new `pill` style rating which is set with the `RATING_TYPES` property.
```

File: UNRELEASED.yaml [hunk 5]
```diff
@@ -1,6 +1,12 @@
 # UNRELEASED:
 
-# How to write a good changelog entry:
+ADDED:
+  - bpk-component-rating:
+    - Rating has been designed to have a fresh look.
```
Comment (shaundon) [inline UNRELEASED.yaml:5]:
```suggestion
    - Rating has been redesigned to have a fresh look.
```

**Pull request merged**